### PR TITLE
Add support for ATtiny 3224/7

### PR DESCRIPTION
**Pull request by lyuboasenov**
Hello,

I've added support for Atttiny 3224/7. I've tested it by creating a door & PIR sensor using ATTiny 3224. I also enabled soft signing for that I had to reduce route size in eeprom because the MCU has 256 b of eeprom.

### CODE REVIEW

**Comment by tekka007** [comment]
Thanks for submitting. A few things need to be fixed beforehand. See here:

https://ci.mysensors.org/job/MySensors/job/MySensors/job/PR-1535/9/The_20Butler_20report/

**Comment by fallberg** [comment]
Found 1 violations:

**Reporter**: Cppcheck
**Rule**: unreadVariable
**Severity**: INFO
**File**: hal/architecture/AVR/MyHwMegaAVR.cpp L395

Variable &#39;count&#39; is assigned a value that is never used.

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a658707407>*

 *<ACCUMULATED-VIOLATIONS>*

File: hal/architecture/AVR/MyHwMegaAVR.cpp [hunk 1]
```diff
@@ -0,0 +1,433 @@
...
+	// wait for pit to interrupt
+	while (bit_is_clear(RTC.PITINTFLAGS, RTC_PI_bp)) {};
+
+	TCA0.SINGLE.CTRLA = 0;                                   /* Enable: disabled */
+
+	RTC.PITCTRLA &= ~(1 << RTC_PITEN_bp);                    /* Enable: disabled */
+
+	sei();
+
+	uint16_t count = TCA0.SINGLE.CNT;
```
Comment (fallberg) [inline hal/architecture/AVR/MyHwMegaAVR.cpp:395]: **Reporter**: Cppcheck
**Rule**: unreadVariable
**Severity**: INFO
**File**: hal/architecture/AVR/MyHwMegaAVR.cpp L395

Variable &#39;count&#39; is assigned a value that is never used.

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a658707407>*

**Comment by tekka007** [comment]
Thank you for your contribution! 👍

**Pull request merged**